### ISSUE TITLE
[BUG] Working Directory for REPL vs. .jl

### ISSUE THREAD

**Issue by versipellis** [JunoLab/Juno.jl#330]
Please search existing issues to avoid duplicates.

## Details
- Atom version: 1.33
- Julia version: 1.1
- OS: Ubuntu 17.10
- Package versions:
  - Atom.jl: 0.8.5
  - julia-client: 0.8.9
  - ink: 0.10.10

### Steps to reproduce
I'm following the basic setup instructions for Genie here: https://github.com/GenieFramework/Genie.jl#genie
Example folder structure being used:
```
/repo/
---/genieapp
```
This code block works in the REPL:
```julia
cd("./genieapp")
using Pkg
pkg"activate ."
using Genie
Genie.loadapp()
```
Trying to run it line-by-line from a `startup.jl` script in `/repo/` throws `could not open file /home/computer/repo/bootstrap.jl`. It appears that when running the code from `startup.jl`, it's not taking into account that the working directory has been changed and is instead running it from the script's location/the Atom project head. Using the `Julia > Working Directory` dropdown menu option does not work either.

**Comment by pfitzseb** [comment]
Right, you need to put that code into `juno_startup.jl`, as per the [FAQ](http://docs.junolab.org/latest/man/faq/#How-do-I-execute-code-on-Juno-startup?-1).

**Comment by versipellis** [comment]
Sorry, I didn't intend to refer to the base Julia's `~/.julia/config/startup.jl` file. I was referring to a separate `startup.jl` file I had created for my process. There is no relation between the two.

Is `juno_startup.jl` the only way to actually change the location that the `.jl` files are executing code from?
